nima_core/storage/temporal_decay.py

```
@@ -188,29 +203,59 @@ def compute_activation(self, node_id: str) -> float:
+        if not scorer_plugins:
+            return base_activation
+
+        # Combine scores from all scorer plugins (weighted average)
+        plugin_scores = []
+        for plugin in scorer_plugins:
+            try:
+                score = plugin.score(node_id, {"base_activation": base_activation})
+                # Ensure score is in valid range
+                score = max(0.0, min(1.0, score))
+                plugin_scores.append(score)
+            except Exception as exc:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)